lib/Stats.js

```diff
@@ -748,13 +748,7 @@ class Stats {
 		if (typeof obj.builtAt === "number") {
 			const builtAtDate = new Date(obj.builtAt);
 			colors.normal("Built at: ");
-			colors.normal(
-				builtAtDate.toLocaleDateString({
-					day: "2-digit",
-					month: "2-digit",
-					year: "numeric"
-				})
-			);
```

Comment:
Looks like this was trying to use `toLocaleDateString(undefined, { day: "2-digit", ... })`, such that it would use the user's locale, spec is [here](https://www.ecma-international.org/ecma-402/1.0/#sec-13.3.2). Unfortunately it looks like ECMA-402 isn't significantly covered by any typescript `--lib` option, nor is there a DT package?

Reply:
This should be in `lib.es5.d.ts`, at least on 2.8.

Reply:
Typescript master only has the ECMA-262 definition, with no parameters.
ECMA-402 amends a bunch of APIs, including this, with internationalisation support.

Reply:
I can be wrong but when I tried, Latest Chrome ignored this parameter.

Reply:
Here is the line in typescript master: https://github.com/Microsoft/TypeScript/blob/ae0ccf340fddb398bfe1ef2e1522624d47ce9b18/src/lib/es5.d.ts#L4120

The call does need to add an undefined for the locale; when I tried this call with chrome the options are ignored when they are the first argument.